docs/shared/node-tutorial/1-code-generation.md

```diff
@@ -18,98 +18,67 @@ In this tutorial you'll create a backend-focused workspace with Nx.
-
-![Our Workspace Requirements](/shared/node-tutorial/requirements-diagram.svg)
+For this tutorial, you'll create an Express API application, a library that the API can reference to handle authentication and a suite of e2e tests.
 
 ## Creating an Nx Workspace
 
 Run the command `npx create-nx-workspace@latest` and when prompted, provide the following responses:
 
 ```{% command="npx create-nx-workspace@latest" path="~" %}
-✔ Choose your style                     · integrated
-✔ What to create in the new workspace   · ts
-✔ Repository name                       · my-products
+✔ Choose your style                     · node-server
+✔ What framework should be used?        · express
+✔ Repository name                       · products-api
 ✔ Enable distributed caching to make your CI faster · Yes
 ```
 
 {% card title="Opting into Nx Cloud" description="You will also be prompted whether to add Nx Cloud to your workspace. We won't address this in this tutorial, but you can see the introduction to Nx Cloud for more details." url="/nx-cloud/intro/what-is-nx-cloud" /%}
 
-## Install the Node Plugin
-
-Open the folder that was created and install the `@nrwl/node` plugin.
-
-```shell
-cd my-products
-npm i -D @nrwl/node
-```
-
-## Add Two Application to Your Workspace
-
-```{% command="nx g @nrwl/node:app products-api" path="~/my-products" %}
->  NX  Generating @nrwl/node:application
-
-CREATE packages/products-api/src/app/.gitkeep
-CREATE packages/products-api/src/assets/.gitkeep
-CREATE packages/products-api/src/main.ts
-CREATE packages/products-api/tsconfig.app.json
-CREATE packages/products-api/tsconfig.json
-CREATE packages/products-api/project.json
-CREATE packages/products-api/.eslintrc.json
-CREATE packages/products-api/jest.config.ts
-CREATE packages/products-api/tsconfig.spec.json
-```
+The `node-server` preset automatically creates a `products-api` application at the root of the workspace and an `e2e` project that runs against it.
 
-Run this command to create your `products-cli` app:
-
-```{% command="npx nx g @nrwl/node:app products-cli" path="~/my-products" %}
->  NX  Generating @nrwl/node:application
-
-CREATE packages/products-cli/src/app/.gitkeep
-CREATE packages/products-cli/src/assets/.gitkeep
-CREATE packages/products-cli/src/main.ts
-CREATE packages/products-cli/tsconfig.app.json
-CREATE packages/products-cli/tsconfig.json
-CREATE packages/products-cli/project.json
-CREATE packages/products-cli/.eslintrc.json
-CREATE packages/products-cli/jest.config.ts
-CREATE packages/products-cli/tsconfig.spec.json
-```
-
-![Nx Generator Syntax](/shared/node-tutorial/generator-syntax.svg)
+{% card title="Framework Options" description="This tutorial uses the `express` framework.  The `node-server` preset also provides starter files for `koa` and `fastify`." url="/nx-cloud/intro/what-is-nx-cloud" /%}
```

Comment:
Here the URL is still pointing to Nx Cloud documentation.

Reply:
Switched this to a callout